src/olympia/devhub/templates/devhub/index.html

```diff
@@ -96,24 +96,26 @@
                         {{ _('Unlisted') }}
                       {% endif %}
                     </p>
-                    {% if item.addon.current_version %}
-                      <p>
-                        <strong>{{ _('Latest Version:') }}</strong>
-                        {{ link_if_listed_else_text(item.addon.current_version,
-                                         item.addon.current_version.version) }}
-                      </p>
-                    {% endif %}
-                    {% with position = item.position %}
-                      {% if position and position.pos and position.total %}
+                    {% if item.addon.is_listed %}
```

Comment:
Same as above, it should really be `{% if item.addon.current_version %}` but I couldn't do that while I had the exception raised in `current_version`. I'll change it back.